src/main/java/org/spongepowered/api/block/BlockType.java

```diff
@@ -121,4 +122,18 @@
      */
     float getEmittedLight();
 
+    /**
+     * Gets if the block is able to be gathered using the silk touch
+     * enchantment.
+     *
+     * @return If the block can be harvested with silk touch
+     */
+    boolean canHarvestWithSilk();
```

Comment:
I think I like simon's idea better. This should be isolated to the SilkTouch enchantment.